integration_tests/suite/test_user_room.py

```diff
@@ -116,6 +117,122 @@ def test_list_by_user_uuid(self, room_1, room_2, room_3):
             ),
         )
 
+    @fixtures.http.room(users=[USER_1])
+    @fixtures.http.room(users=[USER_2])
+    @fixtures.http.room(users=[USER_1, USER_2])
+    def test_list_messages_from_user(self, room1, room2, room3):
+        user1_token = self.asset_cls.create_user_token(user_uuid=USER_1['uuid'])
+
+        self.chatd.set_token(user1_token)
+        # user1 posts a message in the room1
+        self.chatd.rooms.create_message_from_user(
+            room_uuid=room1['uuid'],
+            message_args={'content': 'test message 1'},
+        )
+
+        # user1 posts a message in room3
+        self.chatd.rooms.create_message_from_user(
+            room_uuid=room3['uuid'],
+            message_args={'content': 'test message 2'},
+        )
```

Comment:
useless fixtures, the test want to test the accessibility of the endpoint, not the content